util/symbols.py

```diff
@@ -355,6 +416,18 @@ def retrieve_from_ranges(vram, rom=None):
         return None
 
 
+def mark_c_funcs_as_defined():
+    to_define = set(to_mark_as_defined)
+
+    for symbol in all_symbols:
+        if len(to_define) == 0:
+            return
```

Comment:
```suggestion
    if len(to_define) == 0:
        return
    for symbol in all_symbols:
```

Reply:
to_define gets modified inside the loop, so we have to check its length every loop iteration